project.clj

```diff
@@ -38,9 +38,17 @@
                                   [puppetlabs/trapperkeeper nil :classifier "test"]
                                   [puppetlabs/trapperkeeper-webserver-jetty9]
                                   [puppetlabs/trapperkeeper-webserver-jetty9 nil :classifier "test"]
-                                  [puppetlabs/ring-middleware]]
-                   ;; TK-143, enable SSLv3 for unit tests that exercise SSLv3
-                   :jvm-opts ["-Djava.security.properties=./dev-resources/java.security"]}
+                                  [puppetlabs/ring-middleware]
+                                  [org.bouncycastle/bcpkix-jdk15on]]}
+             :fips {:dependencies [[cheshire]
+                                   [puppetlabs/kitchensink nil :classifier "test"]
```

Comment:
We no longer need to include the `nil` in profile dependencies. The bug that required that was fixed in leiningen 2.8.0. When we bump the travis lein version can/should we bump the `min-lein-version` in this file and remove the `nil`s?

Reply:
Yeah, no worries.